lib/class-wp-theme-json-gutenberg.php

```diff
@@ -790,6 +791,69 @@ public function __construct( $theme_json = array( 'version' => WP_Theme_JSON_Gut
 		}
 	}
 
+	/**
+	 * Unwraps shared block style variations.
+	 *
+	 * @since 6.6.0
+	 *
+	 * {
+	 *   "styles": {
+	 *     "blocks": {
+	 *       "variations": {
+	 *         "blockTypes": [ "core/paragraph", "core/group" ],
+	 *         "color": { "background": "backgroundColor" }
+	 *       }
+	 *     }
+	 *   }
+	 * }
+	 *
+	 * It returns the following output:
+	 *
+	 * {
+	 *   "styles": {
+	 *     "blocks": {
+	 *       "variations": {
+	 *         "core/paragraph": { "color": { "background": "backgroundColor" } },
+	 *         "core/group": { "color": { "background": "backgroundColor" } }
+	 *       }
+	 *     }
+	 *   }
+	 * }
```

Comment:
Similar here, my attempted fix:
```suggestion
	 * {
	 *   "styles": {
	 *     "blocks": {
	 *       "core/paragraph": {
	 *         "variations": {
	 *           "light": { "color": { "background": "backgroundColor" } }
	 *         }
	 *       },
	 *       "core/group": {
	 *         "variations": { 
	 *           "light": { "color": { "background": "backgroundColor" } }
	 *         }
	 *       }
	 *     }
	 *   }
	 * }
```